internal/adapters/arm/database/mssql.go

```diff
@@ -44,6 +44,7 @@ func adaptSecurityAlertPolicies(resource azure.Resource, deployment azure.Deploy
 	for _, policy := range deployment.GetResourcesByType("Microsoft.Sql/servers/securityAlertPolicies") {
 		policies = append(policies, database.SecurityAlertPolicy{
 			Metadata:           policy.Metadata,
+			State:              resource.Properties.GetMapValue("state").AsStringValue("", resource.Metadata),
```

Comment:
if `resource.Properties.GetMapValue("state")` is nil, this will panic. We should check the state first and then use it.

Reply:
Hi @simar7 , TBH I don't know how to make it "right". Do you have any tips ? Thanks